ui/apps/platform/src/types/featureFlag.ts

```diff
@@ -2,12 +2,15 @@
 // However, add strings in alphabetical order to minimize merge conflicts when multiple people add strings.
 // prettier-ignore
 export type FeatureFlagEnvVar =
+    | 'ROX_ACSCS_EMAIL_NOTIFIER'
     | 'ROX_ACTIVE_VULN_MGMT'
     | 'ROX_ADMINISTRATION_EVENTS'
     | 'ROX_CLOUD_CREDENTIALS'
     | 'ROX_COMPLIANCE_ENHANCEMENTS'
     | 'ROX_COMPLIANCE_HIERARCHY_CONTROL_DATA'
     | 'ROX_COMPLIANCE_REPORTING'
+    | 'ROX_MANAGED_CENTRAL'
+    | 'ROX_MOVE_INIT_BUNDLES_UI'
```

Comment:
Is this an intentional return of `ROX_MOVE_INIT_BUNDLES_UI`?

Reply:
oops, definitely not, good catch!